requirements.txt

```diff
@@ -2,6 +2,7 @@ urllib3<1.25
 requests==2.22.0
 pytest<3.0
 pyyaml==5.2
+MarkupSafe==1.1.1
```

Comment:
Do we need this if it's installed separately? I guess for local development setups, although this is supposed to be a temporary workaround...

Reply:
I'm not sure if this is strictly needed, probably not, but I thought it was better to be more explicit than necessary than hoping that the package manager won't try to do again something weird :grin: